map/framework.cpp

```diff
@@ -3209,3 +3209,15 @@ void Framework::VizualizeRoadsInRect(m2::RectD const & rect)
     }
   }, kScale);
 }
+
+vector<MwmSet::MwmId> Framework::GetMwmsByRect(m2::RectD const & rect)
+{
+  vector<MwmSet::MwmId> result;
+  if (!m_infoGetter)
+    return result;
+
+  auto countryId = m_infoGetter->GetRegionCountryId(rect.Center());
```

Comment:
I don't ask you to implement Delaunay triangulation (but it's not so hard as it sounds). From my point of view, KDTree of mwms bounding boxes is good enough.